packages/ui/src/ui/pages/query-tracker/module/query/actions.ts

```diff
@@ -348,6 +366,6 @@ export const toggleShareQuery =
         await dispatch(requestQueriesList());
```

Comment:
What is the reason to reload queries list when we share/unshare a query?

Reply:
This is done in order to display up-to-date data in the ACO column for the list of old queries